pkg/cmd/target/target.go

```diff
@@ -48,6 +48,7 @@ gardenctl target value/that/matches/pattern --control-plane`,
 
 	cmd.AddCommand(NewCmdUnset(f, NewUnsetOptions(ioStreams)))
 	cmd.AddCommand(NewCmdView(f, NewViewOptions(ioStreams)))
+	cmd.AddCommand(NewCmdKubeconfig(f, NewKubeconfigOptions(ioStreams)))
```

Comment:
I think it is better to have this command at top level. It is a command the uses the current target and does something with this information like `ssh` or `k-env` or `p-env`. They are also not subcommands of target.

Reply:
Or we create a new top level `get` command (like kubectl) and add this one as first subcommand. In the future we could add a `target` subcommand and deprecate `target view`.

Reply:
✅